src/rpdk/core/contract/suite/contract_asserts.py

```diff
@@ -85,7 +84,11 @@ def response_does_not_contain_write_only_properties(resource_client, response):
 def response_contains_resource_model_equal_current_model(
     response, current_resource_model
 ):
-    assert response["resourceModel"] == current_resource_model
+    assert (
+        response["resourceModel"] == current_resource_model
+    ), "All properties specified in the request MUST be present in the model \
+        returned, and they MUST match exactly, with the exception of properties\
+             defined as writeOnlyProperties in the resource schema"
```

Comment:
think this error message might be misleading because there should be more exceptions:
[readOnlyProperties, writeOnlyProperties, defaults, insertionOrder, uniqueItems, etc.](https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/#resource-semantics)

Reply:
this been taken from the handler contract agreement